### DISCUSSION TITLE
Numpy conflict

### DISCUSSION THREAD

**Discussion by Divya216** [online-ml/river#1138] [Q&A] [answered]
Hello,
I am working on Google collab

I am getting the below compatible error with numpy when i run the below
from river import datasets
from river import ensemble
from river import evaluate
from river import metrics
from river import tree
from sklearn import linear_model

AttributeError: module 'numpy' has no attribute '_no_nep50_warning'

Version used for river and numpy

river in /usr/local/lib/python3.8/dist-packages (0.14.0)
numpy>=1.23.4 in /usr/local/lib/python3.8/dist-packages (from river) (1.24.1)

while I try the below lines

ValueError: numpy.ufunc size changed, may indicate binary incompatibility. Expected 232 from C header, got 216 from PyObject

Any suggesions?

**Comment by MaxHalford** [comment, marked as answer]
Did you try restarting your notebook after installing River? That works for me: https://colab.research.google.com/drive/10sU2TqbxLQPe4C2ojATWA3EGLJpHp2Cb?usp=sharing

**Reply by Divya216** [reply]
Yes it worked after restarting!! Thanks much :)
